src/transformers/models/beit/modeling_beit.py

```diff
@@ -248,11 +250,7 @@ def __init__(self, config):
 
         self.projection = nn.Conv2d(num_channels, hidden_size, kernel_size=patch_size, stride=patch_size)
 
-    def forward(
-        self,
-        pixel_values: torch.Tensor,
-        position_embedding: Optional[torch.Tensor] = None,
-    ) -> torch.Tensor:
```

Comment:
for consistency with other models `position_embedding` removed from `BeitPatchEmbeddings` and applied in `BeitEmbeddings` module, which is a breaking change, but I suppose `BeitPatchEmbeddings` module is used only as a part of the `BeitEmbeddings`.